src/libraries/System.Private.CoreLib/src/System/Globalization/Normalization.Windows.cs

```diff
@@ -70,7 +71,7 @@ internal static string Normalize(string strInput, NormalizationForm normalizatio
             // to 0 (ERROR_SUCCESS) before executing the calls.
 
             // Guess our buffer size first
-            int iLength = Interop.Normaliz.NormalizeString((int)normalizationForm, strInput, strInput.Length, null, 0);
+            int iLength = Interop.Normaliz.NormalizeString(normalizationForm, strInput, strInput.Length, Span<char>.Empty);
```

Comment:
Can we get rid of this guessing now, and just try to convert it for real?

Reply:
Changed, it's now more similar to what we do on Unix